packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -26,6 +27,42 @@ const BetaWrapper = styled.span`
   padding-top: 0.2rem;
 `;
 
+function UnionParametersRow(props) {
+  const typesToRender = useTypesToRender({
+    property: props.types,
+    apiKey: props.apiKey,
+    isParameter: true,
+  });
+
+  return (
+    <>
+      Can be{' '}
+      {typesToRender.map(({ type }, idx, { length }) => {
```

Comment:
Minor suggestion. We can move the conditional into the body of the span so we have less repeat characters.

```
{typesToRender.map(({ type }, idx, { length }) => <span key={idx}>{length > idx + 1 ? `${type} ,` : `or ${type}`}</span>}
```